src/api/utils/classification.py

```
@@ -73,7 +73,7 @@ def check_if_valid_image(instance):
             if settings.MAX_IMAGE_PIXELS < w * h:
                 raise ValueError(f"Maximum number of pixels is {settings.MAX_IMAGE_PIXELS}.")
         return
-    except (AttributeError, TypeError, IOError, SyntaxError) as _:
+    except (OSError, AttributeError, TypeError, SyntaxError) as _:
```

Flake8 (7.3.0)

[error] local variable '_' is assigned to but never used

(F841)